client/src/components/student/QuizReview.js

```diff
@@ -289,9 +289,9 @@ class QuizReview extends Component {
     return (
         <div>
             <div className="columns">
-                <div className="column">
+                <div className="column is-6">
                 <h2 className="subtitle is-2">{quizAttempt.quiz.title}</h2>
-                    <h2 className="subtitle is-2">Score: {formattedScore} ({this.predictedScore(quizAttempt.quiz.questions.length,predictedConcepts)}% Predicted)</h2>
+                    <h2 className="subtitle is-2">Score: {formattedScore} <span id="notBold">({this.predictedScore(quizAttempt.quiz.questions.length,predictedConcepts)}% Predicted)</span></h2>
```

Comment:
Since there are multiple of these notBold spans on a page, I'll switch this to a class (ID should be unique per page)